package.json

```diff
@@ -11,7 +11,6 @@
     "doctoc": "doctoc --maxlevel 3 --title \"**Table of Contents**\"",
     "test": "nps test",
     "test:ts": "(tsc -p ./tsconfig.json && rimraf test-ts) || rimraf test-ts",
-    "commitmsg": "opt --in commit-msg --exec \"validate-commit-msg\"",
```

Comment:
Don't need this hook anymore because we're not using `validate-commit-msg` anymore